Code/CSharpLINQ/Homework.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace CSharpLINQ
+{
+    public class Homework
+    {
+        public void Task_01()
+        {
+            List<int> numbers = new List<int> { 27, 14, 8, 43, 19, 56, 32, 9, 65, 21 };
+            var divisibleByThree = numbers.Where(number => number % 3 == 0).ToList();
+            Console.WriteLine($"The numbers from the list: [{string.Join(", ", numbers)}]\nthat are divisible by 3 are:  [{string.Join(", ", divisibleByThree)}].");
+        }
+        public void Task_02()
+        {
+            List<int> numbers = new List<int> { 7, -8, -9, 9, 10, -11, -7, 12, -13, 14 };
+            var negativeAndLessThanTen = numbers.Where(number => number < 10).ToList();
+            Console.WriteLine($"The numbers from the list: [{string.Join(", ", numbers)}] \nthat are negative and less than 10 are: [{string.Join(", ", negativeAndLessThanTen)}].");
+        }
+        public void Task_03()
+        {
+            List<string> fruts = new List<string> { "apple", "orange", "pear", "banana", "peach" };
```

Comment:
just a minor comment - use var fruts instead of List<string> fruts :) It will improve readability :)

Reply:
done